sdk/personalizer/Azure.AI.Personalizer/src/Models/DecisionContext.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Azure.AI.Personalizer
+{
+    /// <summary> The Decision Context. </summary>
+    public class DecisionContext
+    {
+        /// <summary> The Decision Context used to serialize an object. </summary>
+        public DecisionContext()
+        {
+        }
+
+        /// <summary> Initializes a new instance of DecisionContext. </summary>
+        /// <param name="rankRequest"> Personalizer Rank Options </param>
+        public DecisionContext(PersonalizerRankOptions rankRequest)
+        {
+            List<string> jsonFeatures = rankRequest.ContextFeatures.Select(f => JsonSerializer.Serialize(f)).ToList();
```

Comment:
[naming] these are the context features, so I think a better name would be contextFeatures

Reply:
I did some refactoring for this method so that the local variable does not exist any more. But I renamed the other places.